tactics/tactics.ml

```diff
@@ -4973,9 +4973,15 @@ module New = struct
   open Locus
 
   let reduce_after_refine =
+    let onhyps =
```

Comment:
Hmm, how did that get there... Probably I messed with the merge, I'll look into it.